main.go

```diff
@@ -282,6 +287,24 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 			} else {
 				s.ChannelMessageSend(m.ChannelID, rps(s, m, rpsChoice))
 			}
+		case botPrefix + "nickchange":
+			st, err := s.UserChannelCreate(m.Author.ID)
+			if err != nil {
+				s.ChannelMessageSend(m.ChannelID, fmt.Sprintf("could not create a private channel for %v", m.Author))
+			}
+			nickArgs := strings.SplitN(m.Content, " ", 4)
+			user := nickArgs[1]
+			nick := nickArgs[2]
+			g, err := getGuild(s, m.ChannelID)
+			if err != nil {
+				s.ChannelMessageSend(st.ID, fmt.Sprintf("an error getting guild: %v", err))
+			}
+			if err := s.GuildMemberNickname(g.ID, user, nick); err != nil {
+				s.ChannelMessageSend(st.ID, fmt.Sprintf("an error occured while changing the nickname of %v to %v: %v", user, nick, err))
+				return
+			}
+			s.ChannelMessageSend(st.ID, fmt.Sprintf("your nick has been changed"))
```

Comment:
As we are changing a different user's nickname, this message doesn't convey the correct idea